frontend/kesaseteli/employer/src/hooks/employments/useValidateEmployment.ts

```diff
@@ -1,21 +1,21 @@
 import useApplicationFormField from 'kesaseteli/employer/hooks/application/useApplicationFormField';
-import noop from 'lodash/noop';
 import React from 'react';
 
 type Options = {
-  onSuccess: () => void
-}
+  onSuccess: () => void;
+};
 
-const useValidateEmployment = (index: number, {onSuccess }: Options = {onSuccess: noop }): () => Promise<void> => {
```

Comment:
`noop` is dropped because eslint nags about the default value in object destruction. Instead check the emptiness on if-clause